Core/DISET/AuthManager.py

```diff
@@ -256,6 +262,26 @@ def getUsername(self, credDict):
       return True
     return False
 
+  def isUserSuspended(self, credDict):
```

Comment:
There's a unit test for this module, and this method should be part of it: https://github.com/DIRACGrid/DIRAC/blob/rel-v6r21/Core/test/Test_AuthManager.py
And BTW that test is in the wrong directory, please move it to the good one since you are it.

Reply:
Added

Reply:
Which directory the Test_... should go ?

Reply:
Inside DIRAC/Core/DISET/test